File: LeetCode/Easy/0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        const int size = nums.size()+1;
        bool *exist = new bool[size]{false};
        vector<int> ret;
        
        for (int i = 0; i < nums.size(); ++i) exist[nums[i]] = true;
        for (int i = 1; i < nums.size() + 1; ++i) {
            if (!exist[i]) ret.push_back(i);
        }
        delete[] exist;
        return ret;
    }
};

File: LeetCode/Easy/0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.go
func findDisappearedNumbers(nums []int) []int {
    exist := make([]bool, len(nums) + 1)
    ret := make([]int, 0)
    for i := 0; i < len(nums); i++ {exist[nums[i]] = true}
    for i := 1; i < len(nums)+1; i++ {
        if !exist[i] {ret = append(ret, i)}
    }
    return ret
}